source/getting-started/advanced-tutorials/fpga-ip-core/index.md

```diff
@@ -0,0 +1,611 @@
+# Tutorial: Custom FPGA IP Core
+
+- **Goal:** Run custom Verilog code in the FPGA.
+- **Complexity:** 5 / 5
+- **Estimated Time:** 90 min
+
+This tutorial goes over the procedure to create and add a custom FPGA IP core in Vivado.
+Step by step instructions are given which walk the user through each part.
+
+## Tutorial Requirements
+
+1. Working AMDC hardware for testing
+2. Completion of the [VSI tutorial](/getting-started/tutorials/vsi/index.md)
+3. Review of the `v1.0` firmware [system architecture documentation](/firmware/arch/system)
+4. Working knowledge of digital logic and Verilog
+    - UW-Madison courses ECE 551/552
+    - Online tutorials, e.g., [1](https://nandland.com/fpga-101/), [2](https://fpgatutorial.com/verilog/), [3](http://www.asic-world.com/verilog/veritut.html), etc
```

Comment:
Maybe mention CS354 and/or CS400, as C knowledge is required for writing drivers?

Reply:
Done